### PR TITLE
Update BAVet126.xml

### PR DESCRIPTION
**Pull request by DariaElagina**
the wrong editor of the entry corrected

### CODE REVIEW

File: VaticanBAV/et/BAVet126.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
             <titleStmt>
                 <title>Catholic Epistles</title>
                 <editor role="generalEditor" key="AB"/>
-                <editor key="DE"/>
+                <editor key="MKr"/>
```
Comment (eu-genia) [inline VaticanBAV/et/BAVet126.xml:10]: If you mean Magda she is MK

Comment (DariaElagina) [reply VaticanBAV/et/BAVet126.xml:10]: I mean Marcin

Comment (PietroLiuzzo) [reply VaticanBAV/et/BAVet126.xml:10]: MKr is Marcin and is correct

Comment (PietroLiuzzo) [reply VaticanBAV/et/BAVet126.xml:10]: list of abbreviation is visible from the team page https://betamasaheft.eu/team.html

**Pull request merged**